var/spack/repos/builtin/packages/pbbam/package.py

```diff
@@ -4,46 +4,43 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
-from spack.pkg.builtin.boost import Boost
 
 
-class Pbbam(CMakePackage):
+class Pbbam(MesonPackage):
     """The pbbam software package provides components to create, query,
     & edit PacBio BAM files and associated indices.
     These components include a core C++ library,
     bindings for additional languages, and command-line utilities."""
 
     homepage = "https://github.com/PacificBiosciences/pbbam"
-    url = "https://github.com/PacificBiosciences/pbbam/archive/0.18.0.tar.gz"
-
+    url = "https://github.com/PacificBiosciences/pbbam/archive/refs/tags/0.18.0.tar.gz"
+    maintainers = ["snehring"]
+
+    version(
+        "2.1.0",
+        sha256="605944f09654d964ce12c31d67e6766dfb1513f730ef5d4b74829b2b84dd464f",
```

Comment:
Confirmed sha256.

Reply:
I tested the 0.18.0 build on fc36 and gcc 12.2 and on rhel7 with gcc 11.2.0. So we _should_ be good.